docs/stacks/hana-scaleout.md

```diff
@@ -18,6 +18,8 @@ One HANA master machine is required.
 
 One or more HANA worker machines are required.
 
```

Comment:
I think this goes in `hana-scaleout-standby.md`.

Reply:
Done. Pushed the change.